org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/handler/ZigBeeThingHandler.java

```diff
@@ -251,6 +252,9 @@ private synchronized void doNodeInitialisation() {
 
         List<Channel> nodeChannels;
 
+        List<ConfigDescriptionParameter> parameters = new ArrayList<ConfigDescriptionParameter>();
+        parameters.addAll(ZigBeeConfigDescriptionParameters.getParameters());
```

Comment:
Why do you add those explicitly here, and not put them directly into the array list constructor?

Reply:
No special reason. But I can change it.